index.html

```diff
@@ -5,8 +5,92 @@
     <title>HTML Academy: Техномарт</title>
   </head>
   <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS, уровень 1</a>».</p>
-
+<header class="header-main">
+<p class="header-logo"><img src="https://via.placeholder.com/111x18" alt="Логотип «Техномарта»" width="111" height="18"></p>
+<!-- Здесь будет форма поиска -->
+<p class="header-bookmarks"><a href="#">Закладки: 0</a></p>
+<p class="header-cart"><a href="#">Корзина: 0</a></p>
+<p class="header-order"><a href="#">Оформить заказ</a></p>
+<p class="header-description">Интернет-магазин<br>
+строительных материалов<br>
+и инструментов для ремонта</p>
+<div class="header-contacts">
+<span class="header-phone">+7 (812) 555-05-55</span>
+<p class="header-address">г. Санкт-Петербург, ул. Б. Конюшенная, д. 19/8</p></div>
+<div class="user-navigation">
+<ul class="user-navigation-unauthorized"
+<li><a href="#">Войти</a></li>
+<li><a href="#">Регистрация</a></li>
+</ul></div>
+<nav class="main-menu">
+<ul>
+<li>Главная</li>
+<li><a href="#">Компания</a></li>
+<li><a href="#">Каталог</a></li>
+<li><a href="#">Новости</a></li>
+<li><a href="#">Спецпредложения</a></li>
+<li><a href="#">Доставка</a></li>
+<li><a href="#">Контакты</a></li>
+</ul>
+</nav>
+</header>
+<main class="container">
+<h1 class="visually-hidden">Интернет-магазин «Техномарт»</h1>
+	<!-- Здесь будут блоки, выделенные красным -->
+	<section class="main-about">
+		<h2>Компания «Техномарт»</h2>
+<p>Настоящий мужской шоппинг начинается здесь! Огромный выбор товаров для ремонта и строительства не оставит равнодушными любителей сэкономить на гастарбайтерах.</p>
+<p>Мы можем доставить ваш товар в самые отдаленные точки России!<br>
+Техномарт работает со многими транспортными компаниями: </p>
+<ul class="companies-list"><li>Деловые линии</li>
+<li>Автотрейдинг</li>
+<li>ЖелДорЭкспедиция</li></ul>
+<a href="#" class="button-more">Подробнее о компании</a>
+	</section>
+	<section class="main-contacts">
+		<h2>Контакты</h2>
+		<p>Вы можете забрать товар сами, <br>
+заехав в наш офис:</p>
+<p class="contacts-map"><a href="https://goo.gl/maps/KQanHvccPNP2" target="_blank"><img src="https://via.placeholder.com/300x158" alt="Карта проезда" width="300" height="158"></a></p>
+<a href="#" class="button-contacts">Заблудились? Напишите нам!</a>
```

Comment:
По ТЗ является кнопкой
```suggestion
<button class="button-contacts">Заблудились? Напишите нам!</button>
```